libgit/id.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libgit
+
+import (
+	"encoding"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/keybase/kbfs/kbfscrypto"
+	"github.com/pkg/errors"
+)
+
+const (
+	// idByteLen is the number of bytes in a git repo ID.
+	idByteLen = 16
+	// idStringLen is the number of characters in the string
+	// representation of a git repo ID.
+	idStringLen = 2 * idByteLen
+
+	idSuffix = 0x2c
```

Comment:
Just to be sure, did you remember to update the list in `keybase/keybase`?

Reply:
Max already did it, yeah.